docs/dev/remote-coding-environments.md

```diff
@@ -0,0 +1,29 @@
+# Remote Coding Environments For Gaffer
+
+Gaffer is now configured for remote coding environemnts such as GitHub
+Codespaces and Gitpod. This addition allows for an easier and faster way to
+contribute to Gaffer with no manual setup or need to download dependices to a
+local machine.
+
+## GitHub Codespaces
+To use GitHub Codespaces simply open the gaffer repsoitory and click the "Code"
+button drop down. You'll see an option labeled "Open with codespaces" Click on
+it.
+
+This will launch Codespaces with all the correct configuration needed to
+contribute to the Gaffer repository.
+
+For more information on GitHub Codespaces click
+[here](https://github.com/features/codespaces).
```

Comment:
```suggestion
This will launch a Codespaces environment with all the configuration needed
to contribute to Gaffer.
See the [GitHub documentation for more information on Codespaces]
(https://github.com/features/codespaces).
```